### PR TITLE
Add Zamphyr

### PR DESCRIPTION
**Pull request by marxo**
Learn to code school fully remote, fully online.

### CODE REVIEW

**Comment by RichardLitt** [comment]
Do I have to know Bosnian to apply? Might be worth mentioning that in the text, if true.

**Comment by marxo** [comment]
@RichardLitt Nope, Serbo-Croatian (Bosnian, Croatian, Serbia) is not a requirement for joining the team, the entire team is at least bilingual, we're all well-versed in English.

**Comment by RichardLitt** [comment]
Cool. Maybe include that on the site? It wasn't clear to me. :)

File: README.md [hunk 1]
```diff
@@ -284,6 +284,7 @@ A curated list of awesome [remote working](https://en.wikipedia.org/wiki/Telecom
   1. [Wikimedia](https://wikimediafoundation.org/wiki/Work_with_us)
   1. [X-Team](https://x-team.com/join/) - Work on incredible web and mobile projects, with extraordinary developers. Unleash your potential. From anywhere. We are X-Team.
   1. [Zapier](https://zapier.com/jobs/)
+  1. [Zamphyr](https://zamphyr.com/posao/) - School 2.0 for learning computer science. Remote-first, 100% JavaScript mostly Meteor.
```
Comment (RichardLitt) [inline README.md]: Move above the line above it, so it is alphabetical.

**Comment by marxo** [comment]
@RichardLitt Thanks for the suggestion. I'll make sure that the next release includes a clear copy regarding that.

**Comment by RichardLitt** [comment]
Merged in 7009161afed116d5fa0fd173a2fcb3d1b6e9bdde. Thank you!

**Pull request closed without merging**